src/tera_main.cc

```diff
@@ -78,6 +77,9 @@ int main(int argc, char** argv) {
         signal(SIGTERM, SignalIntHandler);
         // signal(SIGSEGV, SIG_DFL); // 如果这个被改回SIG_DFL, jvm会core, 不知道为啥...
     }
+    if (g_quit) {
```

Comment:
@elfring  Thanks for the advice but the goto statement is controversial and we try to avoid the usage. A duplicated check is relatively more acceptable.